core/src/main/java/nu/yona/server/messaging/entities/BuddyMessage.java

```diff
@@ -33,6 +33,14 @@
 	private Optional<UUID> senderUserPhotoId;
 	private byte[] senderUserPhotoIdCiphertext;
 
+	@Transient
+	private String firstName;
```

Comment:
Strictly speaking, we don't need first and last name on all subtypes of ``BuddyMessage``, but only on ``BuddyConnectMessage`` (actually its subtypes) and ``BuddyDisconnectMessage``.

It could be addressed by introducing another message base type, e.g. ``BuddyRelationshipMessage``, as a base type for ``BuddyConnectMessage`` and ``BuddyDisconnectMessage``, and put first and last name on that.

What do you think of that?